x-pack/plugin/ccr/qa/security/leader-roles.yml

```diff
@@ -4,7 +4,5 @@ ccruser:
   indices:
     - names: [ 'allowed-index', 'clean-leader', 'forget-leader', 'logs-eu*' ]
       privileges:
-        - monitor
+        - manage
         - read
-        - manage_leader_index
-        - view_index_metadata
```

Comment:
Neither of these privileges are documented as part of CCR. So the documentation is broken for more than just the retention lease renewal action. I added a comment [here](https://github.com/elastic/elasticsearch/issues/61308#issuecomment-1054886981) to track necessary documentation changes.